src/components/OptimizedImage.js

```diff
@@ -54,9 +55,8 @@ export default function OptimizedImage(props) {
           position: 'absolute',
           inset: 0,
           objectFit,
-          width: 'calc(100%)', // width for image rotation
-          height: 'calc(100%)', // height for image rotation
-          transform: `rotate(${rotation}deg)`,
+          width: '100%',
+          height: '98%',
```

Comment:
What was the reason for reducing the height? For me it renders with a gap at the bottom in Edit Grower:
<img width="500" alt="Screenshot 2022-04-23 at 15 16 06" src="https://user-images.githubusercontent.com/5558838/164909788-06dcc7b0-cdd8-4efb-935f-9faba1a25b91.png">

Reply:
Ah, I reduced the height because it was touching the bottom of the container in the Capture Match Image, but I'll put it back to 100% and we'll have to play more with the layout in Capture Match.